tests/php/SubsitesVirtualPageTest.php

```diff
@@ -1,44 +1,62 @@
 <?php
 
+namespace SilverStripe\Subsites\Tests;
+
+use Page;
+use SilverStripe\Assets\File;
+use SilverStripe\Assets\Filesystem;
+use SilverStripe\Assets\Tests\Storage\AssetStoreTest\TestAssetStore;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Control\Director;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\ORM\DB;
+use SilverStripe\Subsites\Model\Subsite;
+use SilverStripe\Subsites\Pages\SubsitesVirtualPage;
+use SilverStripe\Versioned\Versioned;
+
 class SubsitesVirtualPageTest extends BaseSubsiteTest
 {
-	protected static $fixture_file = array(
-		'SubsiteTest.yml',
-		'SubsitesVirtualPageTest.yml',
-	);
+    public static $fixture_file = [
+        'subsites/tests/php/SubsiteTest.yml',
+        'subsites/tests/php/SubsitesVirtualPageTest.yml',
+    ];
+
+    protected $illegalExtensions = [
```

Comment:
`protected static $illegal_extensions = []` with namespaced classes and `@todo` for Translatable